tests/sentry/utils/suspect_resolutions/test_get_suspect_resolutions.py

```diff
@@ -0,0 +1,121 @@
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.models import GroupStatus
+from sentry.testutils import TestCase
+from sentry.utils.suspect_resolutions.get_suspect_resolutions import get_suspect_resolutions
+
+
+class TestProjectIssueCorrelation(TestCase):
```

Comment:
Test classes seems to follow "<NameOfTest>Test", can you refactor this to follow that convention.